pgml-extension/Dockerfile.local

```diff
@@ -11,7 +11,7 @@ RUN cat /etc/apt/sources.list
 RUN apt-get update && apt-get install -y postgresql-pgml-14
 
 # Cache this, quicker
-RUN pip3 install xgboost scikit-learn diptest torch lightgbm transformers datasets sentencepiece sentence_transformers sacremoses sacrebleu rouge
+RUN pip3 install xgboost scikit-learn diptest torch lightgbm transformers datasets sentencepiece sentence_transformers sacremoses sacrebleu rouge protobuf
```

Comment:
we should not need protobuf, what requires that?

Reply:
Finbert model needs protobuf.
```sql
SELECT pgml.transform(
    inputs => ARRAY[
        'Stocks rallied and the British pound gained.', 
        'Stocks making the biggest moves midday: Nvidia, Palantir and more'
    ],
    task => '{"task": "text-classification", 
              "model": "ProsusAI/finbert"
             }'::JSONB
) AS market_sentiment;
```

Reply:
wow... that's... indicative of a whole new level... are they actually calling any rpcs, or are they just serializing in a round about way? I don't really care...

Reply:
I could not reproduce the error I saw before without protobuf on two other machines. Looks like we don't need it.